ColumnSet.js

```diff
@@ -52,6 +53,22 @@ define([
 		return subset;
 	}
 
+	function isRootNode(node, rootNode) {
+		// If we've reached the top-level node for the grid then there is no parent column set.
+		// This guard prevents an error when scroll is initated over some node in the grid that is not a descendant of
+		// a column set. This can happen in a grid that has empty space below its rows (grid is taller than the rows).
+		if (rootNode) {
+			if (node === rootNode) {
+				return true;
+			}
+		}
+		else if (domClass.contains(node, '.dgrid-grid')) {
```

Comment:
This will never be true, because the class will never actually contain `.` on the front of it :) Also, `dgrid` will work just as well as `dgrid-grid`.

This function can also be reduced to a single line (which I'm testing with now on both desktop and Surface):

```js
return (rootNode && node === rootNode) || domClass.contains(node, 'dgrid');
```